src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -398,6 +410,25 @@ internal class RecommendationService(
     log.info("Sent domain event for ${existingRecommendationEntity.data.crn} on manager recall decision made asynchronously for recommendationId $recommendationId")
   }
 
+  private fun sendDeleteRecommendationDomainEvent(
+    recommendationId: Long,
+    existingRecommendationEntity: RecommendationEntity,
+    userId: String?,
+  ) {
+    log.info("About to send domain event for ${existingRecommendationEntity.data.crn} on delete a recommendation for recommendationId $recommendationId")
+    try {
+      sendSpoDeleteRecommendationEvent(
+        crn = existingRecommendationEntity.data.crn,
+        contactOutcome = toDeliusContactOutcome(existingRecommendationEntity.data.spoRecallType).toString(),
+        username = userId ?: MrdTextConstants.EMPTY_STRING,
+      )
+    } catch (ex: Exception) {
+      log.info("Failed to send domain event for ${existingRecommendationEntity.data.crn} on manager recall decision for recommendationId $recommendationId reverting isSentToDelius to false")
```

Comment:
It says here that it's reverting isSentToDelius to false, but it doesn't appear to be reverting it.